app/templates/frameworks/contract_review.html

```diff
@@ -62,7 +62,7 @@
       {% call summary.field() %}
         <p>{{ agreement.signedAgreementDetails.signerName }}</p><p>{{ agreement.signedAgreementDetails.signerRole }}</p>
       {% endcall %}
-      {{ summary.edit_link("Edit", url_for('.signer_details', framework_slug=framework.slug, agreement_id=agreement.id)) }}
+      {{ summary.edit_link("Edit", url_for('.signer_details', framework_slug=framework.slug, agreement_id=agreement.id), hidden_text="Person who signed") }}
```

Comment:
The hidden text probably doesn't capital letters at all, if it's going to end up being shown after the word "Edit" or "Change"?

Reply:
Agree about the caps